core/pom.xml

```diff
@@ -384,6 +384,10 @@
       <artifactId>curator-test</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-minikdc</artifactId>
```

Comment:
Is this only needed in test scope?

Reply:
It's in test scope because in dependency management it's set like that.

Reply:
```
[gaborsomogyi:~/spark] SPARK-29082+ 6s 1 ± mvn dependency:tree | grep kdc
[INFO] +- org.apache.hadoop:hadoop-minikdc:jar:2.7.4:test
[INFO] +- org.apache.hadoop:hadoop-minikdc:jar:2.7.4:test
```

Reply:
In production code not needed.

Reply:
Got it, it's set that way in the parent POM. It might be good to declare that here too just for clarity, or if for some reason the parent doesn't declare as test anymore for some reason.